src/components/KCard/KCard.cy.ts

```diff
@@ -1,92 +1,47 @@
 import { mount } from 'cypress/vue'
 import KCard from '@/components/KCard/KCard.vue'
-import { h } from 'vue'
 
-/**
- * ALL TESTS MUST USE testMode: true
- * We generate unique IDs for reference by aria properties. Test mode strips these out
- * allowing for successful snapshot verification.
- * props: {
- *   testMode: true
- * }
- */
 describe('KCard', () => {
-  it('renders slots when passed', () => {
-    const cardStatusHat = 'Card Status Hat'
-    const cardTitle = 'Card Title'
-    const cardActions = 'Card Actions'
-    const cardBody = 'Card Body'
-    const cardNotifications = 'Card Notifications'
+  it('does not render any content when no props or slots are passed', () => {
```

Comment:
This is misleading - the HTML for the card itself is still rendered

Reply:
Fixed